src/graphql/mutation/createMutationType.js

```diff
@@ -15,13 +15,13 @@ const createMutationType = schema =>
         schema
         .getProcedures()
         .filter(({ isMutation }) => isMutation)
-        .filter(procedure => !procedure.hasTableArg())
```

Comment:
The reason I filtered out procedures with table args is because I wasn’t sure how input for such procedures would work, could you maybe add a test case?